galeracluster/source/notificationcmd.rst

```diff
@@ -53,6 +53,134 @@ The notification command passes with the ``--member`` parameter a list containin
 - **Node Name** Refers to the node name, as you define it for the :ref:`wsrep_node_name <wsrep_node_name>` parameter, in the configuration file.
 - **Incoming Address** Refers to the IP address for client connections, as set for the :ref:`wsrep_node_incoming_address <wsrep_node_incoming_address>` parameter, in the configuration file.
 
+----------------------------------------
+Example Notification Script
+----------------------------------------
+.. _`example-notification-script`:
+
+The particular notification script is the command the node calls in response to replication events.  You can choose the particular command it calls using the :ref:`wsrep_notify_cmd <wsrep_notify_cmd>`.  While you are free to develop your own notification script that meets the particular needs of your application infrastructure, you may find it helpful to consider this example as a starting point.
```

Comment:
A node can call a notification script when a change happens in the membership of the cluster, that is, when nodes join or leave the cluster. The name of the script to call is specified using ...